<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="leftVLayout" stretch="7,3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>订阅内容</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <item>
           <property name="text">
            <string>博客订阅</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新闻订阅</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>喜欢</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学校信息</string>
           </property>
           <item>
            <property name="text">
             <string>软件学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>计算机学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>四川大学</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>活动与讲座</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="leftBottomVLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="IRCToolBtn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>转到IRC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="webToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>转到网页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="releaseToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>我要发布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="shareToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>分享到</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="feedbackToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>我要反馈</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>256</width>
             <height>568</height>
            </rect>
           </property>
           <attribute name="label">
            <string>学院新闻</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>256</width>
             <height>568</height>
            </rect>
           </property>
           <attribute name="label">
            <string>学院通知</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>256</width>
             <height>568</height>
            </rect>
           </property>
           <attribute name="label">
            <string>就业信息</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>256</width>
             <height>568</height>
            </rect>
           </property>
           <attribute name="label">
            <string>学术动态</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>256</width>
             <height>568</height>
            </rect>
           </property>
           <attribute name="label">
            <string>服务资源</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3,0,0,0,0">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>118</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="webEditLine">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>32782</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="aheadToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="backToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="refreshToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="newTabToolBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWebEngineView" name="webView1">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWebEngineView" name="webView2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="importOrExport_2">
     <property name="title">
      <string>导入/导出</string>
     </property>
     <addaction name="importFileAction"/>
     <addaction name="exportFileAction"/>
    </widget>
    <addaction name="newSubscriptionAcion"/>
    <addaction name="newFolderAction"/>
    <addaction name="separator"/>
    <addaction name="importOrExport_2"/>
    <addaction name="separator"/>
    <addaction name="manageCacheAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="markAllReadAction"/>
    <addaction name="separator"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="cutAction"/>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="separator"/>
    <addaction name="undoAction"/>
    <addaction name="doAction"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="browse_menu">
    <property name="title">
     <string>浏览(&amp;B)</string>
    </property>
   </widget>
   <widget class="QMenu" name="favorite_menu">
    <property name="title">
     <string>收藏(&amp;A)</string>
    </property>
   </widget>
   <widget class="QMenu" name="tool_menu">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="view_menu"/>
   <addaction name="browse_menu"/>
   <addaction name="favorite_menu"/>
   <addaction name="tool_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="newSubscriptionAcion">
   <property name="text">
    <string>新建推送</string>
   </property>
  </action>
  <action name="newFolderAction">
   <property name="text">
    <string>新建文件夹</string>
   </property>
  </action>
  <action name="manageCacheAction">
   <property name="text">
    <string>管理缓存</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="markAllReadAction">
   <property name="text">
    <string>标记所有文章为已读</string>
   </property>
  </action>
  <action name="importFileAction">
   <property name="text">
    <string>从本地导入</string>
   </property>
  </action>
  <action name="exportFileAction">
   <property name="text">
    <string>导出到本地</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="doAction">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
